### PR TITLE
Remove contract.parent_id

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:
- Fix balance parse latency alert triggering by increasing threshold to 2 min
- Fix monitor publish errors alert triggering by increasing threshold to 50%
- Remove `contract.parent_id` as its use could result in inaccurate values
- Remove leftover minimal entity logic in `SqlEntityListener`
- Suppress `Fabric8ConfigUtils config-map with name not present in namespace` warnings

**Related issue(s)**:

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-mirror-importer/values.yaml [hunk 1]
```diff
@@ -176,9 +176,9 @@ prometheusRules:
   ImporterBalanceParseLatency:
     annotations:
       description: Averaging {{ $value | humanizeDuration }} trying to parse balance stream files for {{ $labels.namespace }}/{{ $labels.pod }}
-      summary: Took longer than 60s to parse balance stream files
+      summary: Took longer than 2m to parse balance stream files
```
Comment (steven-sheehy) [inline charts/hedera-mirror-importer/values.yaml:179]: Balance parsing times keeps trending upwards averaging about 90s. We need to optimize it but until then increase the threshold

**Review by ijungmann — APPROVED** [review]
LGTM

**Review by Nana-EC — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**